shared/templates/grub2_bootloader_argument/template.py

```diff
@@ -8,4 +8,9 @@ def preprocess(data, lang):
         data["escaped_arg_name_value"] = data["arg_name_value"].replace(".", "\\.")
         # replace . with _, this is used in test / object / state ids
         data["sanitized_arg_name"] = ssg.utils.escape_id(data["arg_name"])
+
+        data['id_suffix'] = ''
+        if 'uefi' in data['_rule_id']:
+            data['id_suffix'] = 'uefi'
+
```

Comment:
This feels a bit hacky. Another approach could be to add the `id_suffix` as another template parameter